src/views/health-checks.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+    <div class="tab">
+        <div v-if="compatible">
+            <h1>Health checks</h1>
+            <p>This is a baby tab. Currently it only checks if all the Custom Elements you're using are imported. Soon we'll add more health checks you can use for your apps.
```

Comment:
I don't know what a baby tab is :) Do you mean "beta feature"?

Reply:
I mean a tab that should be extended with further checks, WDYT we should call it?

Reply:
I think it is fine now, thanks